components/SEOBundle/bundle/Twig/NovaeZSEOExtension.php

```diff
@@ -107,29 +106,20 @@ public function getFallbackedMetaContent(ContentInfo $contentInfo, string $metaN
     /**
      * Compute Metas of the Field thanks to its Content and the Fallback.
      */
-    public function computeMetas(Field $field, ContentInfo $contentInfo): string
+    public function computeMetas(Field $field, Content $content): string
```

Comment:
this is BC break. We don't want to bump a version for that.
Can you remove the type-hint and test it.

If you receive `contentInfo` just convert to a content. And keep the rest.

Add a comment to mention that's to handle a deprecated code.

Reply:
@Plopix done!